src/MockNet/Http/Contents/ObjectContent.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Text;
+using SystemHttpContent = System.Net.Http.HttpContent;
+using SystemStringContent = System.Net.Http.StringContent;
+
+namespace Theorem.MockNet.Http
+{
+    public class ObjectContent : IHttpContent
```

Comment:
Given the media type is fixed to JSON, shouldn't this be called `JsonObjectContent` in that case?

Reply:
Per slack thread leaving the name as ObjectCotent with the default of `application/json` media typ.